src/Role/Commands/CreateRole.php

```diff
@@ -25,6 +22,6 @@ public function __construct(
 
     public function getName(): StringLiteral
     {
-        return $this->name;
+        return new StringLiteral($this->name);
```

Comment:
This method is only used twice, once in real code and once in a test. So it's a opportunity to remove `StringLiteral` from this file

Reply:
Done, I could take it further but it cascades.